platform.js

```diff
@@ -95,12 +95,23 @@ module.exports = {
       }, milliseconds);
     });
   },
-  startProcess(command, args) {
-    childProcess.spawn(command, args, {
-      cwd: path.dirname(command),
-      stdio: "ignore",
-      detached: true
-    }).unref();
+  startProcess(command, args, tries) {
+    if (typeof tries === "undefined") {tries = 3;}
+    if (tries <= 0) {return;}
+    tries -= 1;
+
+    try {
+      childProcess.spawn(command, args, {
+        cwd: path.dirname(command),
+        stdio: "ignore",
+        detached: true
+      }).unref();
+    } catch(err) {
+      if (tries === 0) {throw err;}
```

Comment:
@tejohnso The equality here is too strict. I'm concerned we might receive something like `1.1` as a parameter `tries`, and then we'd be in a promise that never resolves or rejects.

How about `<=`?